packages/connect/src/api/cardano/cardanoOutputs.ts

```diff
@@ -1,30 +1,44 @@
 // origin: https://github.com/trezor/connect/blob/develop/src/js/core/methods/helpers/cardanoOutputs.js
 
+/* eslint-disable no-restricted-syntax */
```

Comment:
Because of the `for (const assetGroup of tokenBundle)` syntax. Eslint doesn't like that and I suppose it prefers `await Promise.awaitAll(tokenBundle.map(async ...))` (I didn't check the syntax is correct) but I think the order isn't guaranteed in that case so it wouldn't work for us. And AFAIK async foreach doesn't exist. The same is used in `cardanoSignTransaction.ts` btw.

I'll add a comment here (and in `cardanoSignTransaction.ts`) to make it clear.

Reply:
Done in https://github.com/vacuumlabs/trezor-suite/pull/1/commits/58062096a0b3ab7b5e023ad1e11789368ceba375.